### ISSUE TITLE
"AFL LLVM runtime is not built with Rust ..." error reported even if afl is built with the same rust version (afl 0.12.13)

### ISSUE THREAD

**Issue by taiki-e** [rust-fuzz/afl.rs#299]
My project uses GitHub workflow like the following to check that fuzzing targets can be built.
This worked until afl 0.12.12, but it no longer works in afl 0.12.13.

```yaml
  fuzz:
    runs-on: ubuntu-latest
    steps:
      - uses: actions/checkout@v3
      - name: Install Rust
        run: rustup toolchain add nightly --no-self-update && rustup default nightly
      - run: cargo install afl --debug --locked
      - run: cargo afl build
        working-directory: fuzz/afl
```

afl 0.12.13 reports the following error:

```
AFL LLVM runtime is not built with Rust rustc-1.69.0-nightly-8996ea9, run `cargo install --force afl` to build it.
```

Howver, they are built with the same rust version.

full workflow: https://github.com/taiki-e/parse-changelog/blob/3af71410252899d73a6440ca87525a9bc57c3049/.github/workflows/ci.yml#L133-L148
full ci log: https://github.com/taiki-e/parse-changelog/actions/runs/4139963843/jobs/7158133724

**Comment by smoelius** [comment]
Thanks very much for the bug report, @taiki-e. For now, I yanked version 0.12.13. I will look at this more closely when I have more time.

**Comment by smoelius** [comment]
Actually, I think this is fixed by 0.12.14.

(I had initially thought not, but I think there was my there was an error in my experiment.)

**Comment by taiki-e** [comment]
Thanks! I confirmed that this was fixed by 0.12.14 (https://github.com/taiki-e/parse-changelog/actions/runs/4149047413/jobs/7177650586).

**Comment by smoelius** [comment]
Thank you again for the bug report.
